packages/devtools_app/lib/src/inspector/inspector_screen.dart

```diff
@@ -160,6 +168,39 @@ class InspectorScreenBodyState extends State<InspectorScreenBody>
     );
   }
 
+  Widget _buildSummaryTreeColumn() {
+    final errors = _errors
+        ?.map((key, value) => MapEntry(key, value as InspectableWidgetError));
+    final errorList = errors?.values?.toList() ?? [];
+
+    return OutlineDecoration(
+      child: Stack(
+        children: [
+          InspectorTree(
+            controller: summaryTreeController,
+            isSummaryTree: true,
+            widgetErrors: errors,
+          ),
+          if (errors.isNotEmpty)
```

Comment:
should this be checking for `errorList.isNotEmpty` to make sure nothing gets out of sync?

Reply:
This list is gone now, as the ErrorNavigator takes the map since it handles the selectedNodeChanged code.